tasks/build.py

```diff
@@ -157,6 +157,7 @@ def _build_jekyll(ctx, branch, owner, repository, site_prefix,
     Builds the cloned site with Jekyll
     '''
     # Add baseurl, branch, and the custom config to _config.yml
+<<<<<<< Updated upstream
```

Comment:
Looks like some git artifacts snuck in.

Reply:
dangit how did I do that o_O

Reply:
rofl I forgot to hit save before committing